CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ changes in the following format: PR #1234***
 - Very old instruments relying on QuickForm may have issues due to code changes (PR #4928)
 - Unix user permissions have been updated which may affect access to files. New
 documentation for file permissions has been added to the README.md file (PR #5323)
+- Addition of a new `account_request_date` in `users` table that will be used when
+requesting a new account and will be displayed in the User Accounts module (PR #6191)
```

Comment:
This is in the wrong section of the CHANGELOG. It's not going into the 23.0 release..

Reply:
oooooooooh, oupsie... 

@driusan should be fixed in the last commit. Sorry about that...